### PR TITLE
implemented metamask sdk

### PR DESCRIPTION
**Pull request by meowyx**
Here's what I did:

- implemented metamask sdk
- connect wallet with metamask functionality is accessible
- user's wallet address is displayed
- also disconnect function

### CODE REVIEW

**Comment by httpJunkie** [comment]
I think we need to create some top margin or padding in order to make the address of the loged in metamask wallet more aligned vertically, as well the dosconnect button should have a similar style as the green button to login.

Otherwise, I think this is a good start.

File: apps/web/components/Navbar.tsx [hunk 1]
```diff
@@ -1,32 +1,82 @@
...
+import { useSDK, MetaMaskProvider } from "@metamask/sdk-react";
+import { formatAddress } from "./../lib/utils";
+
+export const ConnectWalletButton = () => {
+  const [account, setAccount] = useState<string | undefined>();
+  const { sdk, connected, connecting } = useSDK();
+
+  const connect = async () => {
+    try {
+      const metaMaskAccount = (await sdk?.connect()) as string;
```
Comment (chin-flags) [inline apps/web/components/Navbar.tsx:21]: is this type casting to `string` correct ?
isn't `await sdk?.connect()` return an array of accounts ?

File: apps/web/components/Navbar.tsx [hunk 2]
```diff
@@ -1,32 +1,82 @@
...
 
 export const NavBar = () => {
+  const host =
+    typeof window !== "undefined" ? window.location.host : "defaultHost";
+
+  const sdkOptions = {
+    logging: { developerMode: false },
+    checkInstallationImmediately: false,
+    dappMetadata: {
+      name: "Demo React App",
```
Comment (chin-flags) [inline apps/web/components/Navbar.tsx:61]: we could use `EventSea`

File: apps/web/components/Navbar.tsx (hunk 1 above)
Comment (meowyx) [reply apps/web/components/Navbar.tsx:21]: yeah it does but like one wallet can have multiple accounts I believe.
@chin-flags

File: apps/web/components/Navbar.tsx (hunk 2 above)
Comment (meowyx) [reply apps/web/components/Navbar.tsx:61]: Will change that. Thank you @chin-flags

File: apps/web/components/Navbar.tsx (hunk 1 above)
Comment (meowyx) [reply apps/web/components/Navbar.tsx:21]: oh okay. I got it now. Thank you so muchh. @chin-flags

**Review by chin-flags — APPROVED** [review]

**Pull request merged**